duxy/Day2/ej1.java

```diff
@@ -0,0 +1,37 @@
+import java.io.*;
+class ej1 {
+    public static void main(String[] args) {  
+        BufferedReader br = null;
+        try {
+            br = new BufferedReader(new FileReader("input.txt"));
+            String line;
+            int suma = 0;
+            while ((line = br.readLine()) != null) {                
+                String[] parts = line.split("\t");
+                int max = Integer.MIN_VALUE;
+                int min = Integer.MAX_VALUE;
+                for(String number : parts){
+                    int nu_in = Integer.parseInt(number);
+                    if(nu_in > max){
+                       max = nu_in;
+                    }
+                    if(nu_in < min){
+                       min = nu_in;
+                    }
+                }                
+                suma = suma + (max - min);
+            }
+            System.out.print("The result is: " + suma);
```

Comment:
`The result is: + suma? `  please don't write code in spanish!!! never more!! ❌ 🔥 💣 💥  *BUT  IT'S WORST TO MIX SPANISH AND ENGLISH*